purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1899,6 +1898,59 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             }
         }
 
+        /**
+         * Check if billing is supported for the current Play user (meaning IN-APP purchases are supported)
+         * and optionally, whether a specified feature type is supported. This method is asynchronous since
+         * it requires a connected BillingClient.
+         * @param context A context object that will be used to connect to the billing client
+         * @param feature An optional feature type to check for support. Must be one of [BillingFeature]
+         *                  If null, no specific feature support will be checked.
+         * @param callback Callback that will be notified when the check is complete.
+         */
+        @JvmStatic
+        fun canMakePayments(
+            context: Context,
+            feature: BillingFeature?,
```

Comment:
PR in draft because I plan to change this to a list of features. Otherwise the rest should be reviewable